app/components/dashboard_page_component.html.erb

```diff
@@ -25,7 +25,7 @@
   <% if nav_tabs? %>
     <!-- Nav tabs -->
     <div class="row">
-      <div class="col-12 border-bottom">
```

Comment:
There are two bottom borders on the staff pets show page.

Reply:
Wow! Not sure how to caught that. Nice find.